docs/index.md

```diff
@@ -136,10 +137,10 @@ func main() {
 		count  int
 		counts []counted
 	)
-	for scan.Scan(ctx, &token, &count) {
+	for scanner.Scan(ctx, &token, &count) {
 		counts = append(counts, counted{token, count})
 	}
-	if err := scan.Err(); err != nil {
+	if err := scanner.Err(); err != nil {
 		log.Fatal(err)
 	}
 
```

Comment:
add a slice.Discard() into the example?

Reply:
I think we'll try to keep this "getting started" program minimal, but I like the idea of having examples, so I made #80.